roles/blackmagic/defaults/main.yml

```diff
@@ -5,8 +5,18 @@
 # https://www.blackmagicdesign.com/support/family/capture-and-playback
 # Latest Downloads
 # (Linux)
-# Desktop Video 10.9.3 (July 16 2017)
-# Blackmagic_Desktop_Video_Linux_10.9.3.tar.gz
+# Desktop Video 10.9.7 (Sep 7 2017)
+# Linux
+# Bottom of the form is "Download Only"
+# Blackmagic_Desktop_Video_Linux_10.9.7.tar.gz
+# cd /srv/pxe/bm
+# tar xfv Blackmagic_Desktop_Video_Linux_10.9.7.tar.gz
+#
+# to find dkms_version
+# ls /usr/src/blackmagic*
+# blackmagic-10.9.7a2/    blackmagic-io-10.9.7a2/
+# I guess :p
+#
```

Comment:
This all seems rather unrelated.

Reply:
it is tips on how to get the driver from bm.com site.
  and figure out the "10.9.7a2" value for 
dkms_version: 10.8.4a4

I would expect there is a better way, but I don't know what it is.

Reply:
I mean unrelated to the rest of this PR

Reply:
(hmm, email said I could reply.. but I don't see it, so here it is again)

ah sure. 

I bumped into it trying to test everything.   

Should I try to put it into it;s own .. branch?